From 9b13ef61756ea09f5b8781e8e3443caff42b2b45 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ball=C3=B3=20Gy=C3=B6rgy?= <ballogyor@gmail.com>
Date: Wed, 24 Jan 2024 18:51:51 +0100
Subject: [PATCH] theme: improve screensaver style

- Increase the transparency of the top panel.
- Round the corners of the unlock dialog.
---
 data/theme/common.css | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/data/theme/common.css b/data/theme/common.css
index 2193a87..5def8f4 100644
--- a/data/theme/common.css
+++ b/data/theme/common.css
@@ -1,7 +1,7 @@
 gf-screensaver-panel {
   font-weight: bold;
   padding: 4px 10px;
-  background: rgba (0, 0, 0, 0.8);
+  background: rgba (0, 0, 0, 0.3);
   color: #cccccc;
 }
 
@@ -10,6 +10,11 @@ gf-screensaver-panel label {
   color: #cccccc;
 }
 
+gf-unlock-dialog,
+gf-unlock-dialog frame border {
+  border-radius: 30px;
+}
+
 gf-popup-window {
   border-radius: 10px;
 }
-- 
2.44.0

